### PR TITLE
Added Green on Dark

### PR DESCRIPTION
**Pull request by ELChris414**
I have zero idea if this works as there is literally no way for me to test it. Please somebody with a mac and Xcode compile it and tell me if it works! Thanks

Suggestion from this by: sebastianpc#4420 on Discord

### CODE REVIEW

File: app/UserPreferences.m [hunk 1]
```diff
@@ -148,9 +148,11 @@ - (BOOL)isEqual:(id)object {
 
 NSDictionary<NSString *, Theme *> *presetThemes;
 + (void)initialize {
-    presetThemes = @{@"Light": [self _themeWithForegroundColor:UIColor.blackColor
+    presetThemes = @{@"Light":         [self _themeWithForegroundColor:UIColor.blackColor
```
Comment (tbodt) [inline app/UserPreferences.m]: but the :s aren’t aligned anymore

Comment (tbodt) [reply app/UserPreferences.m]: (in the method call I mean)

Comment (ELChris414) [reply app/UserPreferences.m]: I aligned the [ instead:
image
anyways in the current file the [ are aligned

Comment (tbodt) [reply app/UserPreferences.m]: weren’t they both aligned initially?

**Comment by tbodt** [comment]
smooth

**Pull request merged**